checkov/terraform/tf_parser.py

```diff
@@ -699,13 +699,15 @@ def load_or_die_quietly(
     file: str | Path | os.DirEntry[str], parsing_errors: dict[str, Exception], clean_definitions: bool = True
 ) -> Optional[_Hcl2Payload]:
     """
-Load JSON or HCL, depending on filename.
+    Load JSON or HCL, depending on filename.
     :return: None if the file can't be loaded
     """
-
     file_path = os.fspath(file)
     file_name = os.path.basename(file_path)
 
+    if file_name.endswith('.tfvars'):
```

Comment:
can you explain what was the specific issue + doc it in the PR description

Reply:
We are filtering out some kind of static vars, but we don't what to do it for tfvars.